pyproject.toml

```
@@ -8,3 +8,9 @@ requires = ["setuptools",
             "Cython>=0.29.21,<3.0"]
 
 build-backend = 'setuptools.build_meta'
+
+[tool.cibuildwheel]
+# Skip pypy on mac due to numpy/accelerate issues
+skip = "pp37-macosx_x86_64"
+test-requires = "pytest scipy"
+test-command = "pytest $(dirname $(python -c "import astroscrappy; print(astroscrappy.__file__)"))"
```

Oops, this should be better :
```suggestion
test-command = "pytest $(dirname $(python -c 'import astroscrappy; print(astroscrappy.__file__)'))"
```